backend/spotihue/oauth.py

```diff
@@ -50,8 +50,13 @@ def _get_auth_response_local_server(self, redirect_port: int) -> str:
             str: auth code from Spotify.
         """
         server = start_local_http_server(redirect_port)
-        self._open_auth_url()
+        logger.info(f'Listening on port {redirect_port} for Spotify callback...')
+
+        if open_browser:
+            self._open_auth_url()
+
         server.handle_request()
+        logger.info(f'Callback received!')
```

Comment:
Don't think you need the f string here?

Reply:
good catch - removed this f-string plus a couple others.